packages/app/migrations/1622003137738_alter_table_public_integration_add_column_key/up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE "public"."integration" ADD COLUMN "key" text NULL;
+ALTER TABLE "public"."integration" ADD CONSTRAINT "integration_key_key" UNIQUE ("key");
+
+-- There are already DNS records using the integration name, so need to keep those working
```

Comment:
Which DNS records are you referring to?

Reply:
Don't we have a UUID PK for integrations?

Reply:
ah my bad with the comment. this is for use in k8s pod names rather than using the UUID. Like the same named and trigger populated field on Tenant it's not a PK.

Reply:
Gotcha, because we need to scrape a consistent metrics endpoint on the Exporters 👍 makes sense